src/react-chunks.mjs

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2019-2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useRef, useState } from 'react';
+import rmanifest from 'route-manifest';
+import { listen } from './quicklink';
```

Comment:
Just to check: we don't need to specify here other version of the library rather than ./quicklink?
Asking in case the spa version ends up being installed on a different package?

// @addyosmani @anton-karlovskiy

Reply:
My current thinking is that everything inside the dist directory will get published to npm. This should allow this file to just reference the local (chunks) build of Quicklink.

I think that this would be more straightforward than introducing a secondary package that we publish?